app/views/includes/online-tasks.nunjucks

```diff
@@ -1,14 +1,19 @@
 {% if gp.onlineTasks %}
-<ul class="link-list">
-{% if gp.onlineTasks.bookOnlineLink %}
-  <li><a class="gp-book-online" href="{{gp.onlineTasks.bookOnlineLink}}">Book a GP appointment</a></li>
-{% endif %}
-{% if gp.onlineTasks.repeatPrescriptionOnlineLink %}
-  <li><a class="repeat-prescription-online" href="{{gp.onlineTasks.repeatPrescriptionOnlineLink}}">Order a repeat prescription</a></li>
-{% endif %}
-{% if gp.onlineTasks.codedRecordsOnlineLink %}
-  <li><a class="coded-records-online" href="{{gp.onlineTasks.codedRecordsOnlineLink}}">See your health records</a></li>
-{% endif %}
+<ul class="link-list grid-row">
+  {% if gp.onlineTasks.bookOnlineLink %}
+  <li class="column--one-third">
```

Comment:
Sure, if you can write a rule in the linting I'll happily adhere to the automated checks. Otherwise I'm turning a blind eye i.e. I'm not changing it ;)